plugins/woocommerce-blocks/assets/js/data/cart/thunks.ts

```diff
@@ -75,7 +82,316 @@ export const receiveError =
 		dispatch.setErrorData( response );
 	};
 
+/**
```

Comment:
These thunks have just moved from actions.ts. I was going to add some new thunks but took different approach to persistence layer.